<launch>
    <node name="canbus_driver" pkg="autolabor_canbus_driver" type="canbus_driver" output="screen">
        <param name="port_name" value="/dev/ttyUSB0" type="string" />
    </node>

    <node name="pm1_driver" pkg="autolabor_canbus_driver" type="pm1_driver" output="screen" >
      <!--param name="optimize_limit" value="0.75" />
      <param name="smooth_coefficient" value="0.2" /-->
    </node>

    <node name="joy" pkg="joy" type="joy_node" />

    <node name="joy_to_twist" pkg="joy_to_twist" type="joy_to_twist">
      <param name="linear_min" value="0.25" />
      <param name="linear_max" value="2.0" />
      <param name="linear_step" value="0.25" />

      <param name="angular_min" value="0.5" />
      <param name="angular_max" value="2.0" />
      <param name="angular_step" value="0.5" />
    </node>
</launch>
